problem_builder/slider.py

```diff
@@ -172,3 +172,10 @@ def validate_field_data(self, validation, data):
         Validate this block's field data.
         """
         super(SliderBlock, self).validate_field_data(validation, data)
+
+    def student_view_data(self):
```

Comment:
@itsjeyd So my question is, why isn't `SliderBlock` an eligible block type?  And `MRQBlock` too for that matter?  But I didn't have enough time to look into that..

I'm happy to remove this change though.